scripts/generate_snapshots.py

```
@@ -18,12 +18,17 @@
 import shutil
 import sys
 import warnings
+from concurrent.futures import ProcessPoolExecutor
 from pathlib import Path
+from time import perf_counter
 
 sys.path.append(f"{Path.cwd()}")  # Make holidays visible.
-
-import holidays  # noqa: E402
-from holidays import list_supported_countries, list_supported_financial  # noqa: E402
+from holidays import (  # noqa: E402
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)